example/src/screens/HomeScreen.tsx

```
@@ -0,0 +1,493 @@
+import { C } from "../theme";
+import type { BrowseBusy, Busy, SpotifyProfile, TransportBusy } from "../types";
+import { formatAccountTier, formatSpotifyError, sleep } from "../utils/format";
+
+export function HomeScreen() {
+  const connectionState = useConnectionState();
+  const currentTrack = useCurrentTrack();
+  const isPlaying = useIsPlaying();
+  const playbackPositionMs = usePlaybackPosition();
+  const [positionAnchor, setPositionAnchor] = useState<{ positionMs: number; capturedAt: number }>({
+    positionMs: 0,
+    capturedAt: Date.now(),
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/example/src/screens/HomeScreen.tsx:54:17
  52 |   const [positionAnchor, setPositionAnchor] = useState<{ positionMs: number; capturedAt: number }>({
  53 |     positionMs: 0,
> 54 |     capturedAt: Date.now(),
     |                 ^^^^^^^^^^ Cannot call impure function
  55 |   });
  56 |   const [clockTick, setClockTick] = useState(0);
  57 |

(react-hooks/purity)